source/tools/generateContractInterfaces.ts

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+import { ContractCompiler } from "../libraries/ContractCompiler";
+import { Configuration } from '../libraries/Configuration';
+import { ContractInterfaceGenerator } from '../libraries/ContractInterfacesGenerator';
+
+async function doWork(): Promise<void> {
+
+    const configuration = await Configuration.create();
+
+    const compiler: ContractCompiler = new ContractCompiler(configuration);
+
+    const interfacesGenerator: ContractInterfaceGenerator = new ContractInterfaceGenerator(configuration, compiler);
+
+    interfacesGenerator.generateContractInterfaces();
```

Comment:
Style, for such a short function, it feels like the spaces between each line are unnecessary.  Also, the space on first line of the function doesn't align with our style elsewhere.

Reply:
Will fix